Update to use system-installed spdlog

polmoorx opened the pull request:
Update to use system-installed spdlog


#### Merge Checklist  <!-- REQUIRED -->
**All** boxes should be checked before merging the PR
- [x] The changes in the PR have been built and tested
- [] cgmanifest file has been updated if required
- [] Ready to merge

#### Description <!-- REQUIRED -->
<!-- Please include a summary of the changes and the related issue. List any dependencies that are required for this change. -->
Modified the build to use the spdlog-devel package already available
in the system. This reduces duplication and ensures consistent spdlog
updates across the system.
<!-- Fixes # (issue) -->

#### Any Newly Introduced Dependencies
<!-- Please describe any newly introduced 3rd party dependencies in this change. List their name, license information and how they are used in the project. -->
No
#### How Has This Been Tested? <!-- REQUIRED -->
<!-- Please describe the tests that you ran to verify your changes. Provide instructions so we can reproduce. Please also list any relevant details for your test configuration. -->
Manually tested

anujm1 reviewed (changes requested):
level-zero should use system installed spdlog

polmoorx commented:
> level-zero should use system installed spdlog

Thank you for your valuable suggestion. We have updated the .spec file to use the system-installed spdlog.

To ensure nothing is broken, we ran the package test for intel-level-zero, and the build completed successfully

polmoorx committed:
Update to use system-installed spdlog

Modified the build to use the spdlog-devel package already available
in the system. This reduces duplication and ensures consistent spdlog
updates across the system.

Signed-off-by: Polmoorx Shiva Kumar <polmoorx.shiva.kumar@intel.com>
